tests/toolchain/gcc5_fortran_compilation.pm

```diff
@@ -27,7 +28,10 @@ sub run {
     script_run 'tar jxf fcvs21_f95.tar.bz2';
     script_run 'cp FM923.DAT fcvs21_f95/';
     script_run 'pushd fcvs21_f95';
-    script_run "sed -i 's/g77/gfortran-5/g' driver_*";
+    # pattern is not available for sle-15 so
+    # fortran is installed via package and not fixed to a specific version
```

Comment:
don't just accept anything as given. We are QA, not test monkeys :) So for me that is a bug